config/file_resolver.go

```diff
@@ -42,8 +42,11 @@ func NewRelativeResolver(paths ...string) FileResolver {
 
 	copy(pathList, paths)
 
-	// add the current cwd
-	if cwd, err := os.Getwd(); err == nil {
+	if appRoot := os.Getenv(EnvironmentPrefix() + _root); appRoot != "" {
```

Comment:
APP_ROOT sounds good, since it's one thing to rule them all, rather than environment/company specific.